src/app/static-pages/team-page/member-filter/member-filter.component.scss

```diff
@@ -0,0 +1,56 @@
+.filter {
+    text-align: right;
+    h4, i{
+        color: #160637;
+        cursor: pointer;
+        display: inline;
+    }
+    p{
+        color: #8989A2;
+        cursor: pointer;
+        display: inline;
+    }
+}
+.filter-content{
+    background-color: #F8F8F8;
```

Comment:
Die Farben sollten eigentlich auch im colors.scss Stylesheet als Variablen vorhanden sein.
Wenn du das noch importieren könntest und die Variablen nutzt, wäre es glaube ich besser für zukünftige Anpassungen am Style.

Farben die es in der Datei noch nicht gibt, kannst du gerne dort ergänzen.